.github/workflows/reusable-cask-checks.yml

```diff
@@ -0,0 +1,34 @@
+name: Reusable Brew Cask Checks
+
+on:
+  workflow_call:
+    inputs:
+      jdk-version:
+        description: 'Zulu JDK version to install, e.g. jdk11, jdk17, mc (special one for Mission Control), etc.'
+        required: true
+        type: string
```

Comment:
Everything works well now, workflows are reused via `workflow_call` instead of 
a composite action.

- Extra checkout steps are removed in dc5e2e8de33cd56deabed52c87a71fc27bd6eb82.
- Reuse stuffs for missing control as well in 85148eb6fd057a809850e2d99293ab398d354a74.